src/TLuaInterpreter.cpp

```diff
@@ -13723,3 +13439,19 @@ int TLuaInterpreter::startPermSubstringTrigger(const QString & name, const QStri
 
 }
 
+static int host_key = 0;
+
+static void store_host_in_lua_state(lua_State* L, Host* h) {
```

Comment:
:thinking: Would it be applicable to store/access any other data like this? What are the pros and cons of this technique?

If it works (and you must think it does) then :+1:, it is just that the Lua-fu is not string in me so I cannot really judge... :wink: